crates/gitql-ast/src/function.rs

```diff
@@ -136,3 +144,12 @@ fn text_right_trim(inputs: Vec<Value>) -> Value {
 fn text_len(inputs: Vec<Value>) -> Value {
     return Value::Number(inputs[0].as_text().len() as i64);
 }
+
+fn text_ascii(inputs: Vec<Value>) -> Value {
```

Comment:
I think we can check if string is empty before collecting it as vector, also we can get first char without converting to vector for example

```rs
let text = inputs[0].as_text();
if text.is_empty() {
    return Value::Number(0);
}
return Value::Number(text.chars().nth(0).unwrap() as i64);
```